docs/getting-started/products/yvaults/v3.md

```diff
@@ -0,0 +1,537 @@
+
+
+#### Shutting down the vault
+In an emergency the EMERGENCY_MANAGER can shutdown the vault
+
+This will also give the EMERGENCY_MANAGER the DEBT_MANAGER roles as well so funds can start to be returned from the strategies
+
+## Strategy Minimum API
+Strategies are completely independent smart contracts that can be implemented following the proposed template or in any other way.
+
+In any case, to be compatible with the vault, they need to implement the following functions, which are a subset of ERC4626 vaults: 
+- asset(): view returning underlying asset
+- totalAssets(): view returning current amount of assets. It can include rewards valued in `asset` ¡
+- maxDeposit(address): view returning the amount max that the strategy can take safely
+- deposit(assets, receiver): deposits `assets` amount of tokens into the strategy. it can be restricted to vault only or be open
+- maxWithdraw(address): view returning how many asset can the vault take from the vault at any given point in time
+- withdraw(assets, receiver, owner): withdraws `assets` amount of tokens from the strategy
+- redeem(shares, receiver, owner): redeems `shares` of the strategy for the underlying asset.
+- balanceOf(address): return the number of shares of the strategy that the address has
+- convertToAssets(shares: uint256): Converts `shares` into the corresponding amount of asset.
+- convertToShares(assets: uint256): Converts `assets` into the corresponding amount of shares.
+- previewWithdraw(assets: uint256): Converst `assets` into the corresponding amount of shares rounding up.
+
+This means that the vault can deposit into any ERC4626 vault but also that a non-compliant strategy can be implemented provided that these functions have been implemented (even in a non ERC4626 compliant way). 
+
+## ERC4626 compliance
+Vault Shares are ERC4626 compliant. 
+
+The most important implication is that `withdraw` and `redeem` functions as presented in ERC4626, with the ability to add two additional non-standard options.
+
+1. max_loss: The amount in basis points that the withdrawer will accept as a loss. I.E. 100 = 1% loss accepted.
+2. strategies: This is an array of strategies to use as the withdrawal queue instead of the default queue.
+
+### Shutdown mode
+In the case the current roles stop fulfilling their responsibilities or something else happens, the EMERGENCY_MANAGER can shutdown the vault.
+
+The shutdown mode should be the last option in an emergency as it is irreversible. 
+
+### Deposits
+_Light emergency_: Deposits can be paused by setting depositLimit to 0
+
+_Shutdown mode_: Deposits are not allowed
+
+### Withdrawals
+Withdrawals can't be paused under any circumstance
+
+### Accounting
+Shutdown mode does not affect accounting
+
+### Debt rebalance
+_Light emergency_: Setting minimumTotalIdle to MAX_UINT256 will result in the vault requesting the debt back from strategies. This would stop new strategies from getting funded too, as the vault prioritizes minimumTotalIdle
```

Comment:
```suggestion
`_Light emergency_`: Setting minimumTotalIdle to MAX_UINT256 will result in the vault requesting the debt back from strategies. This would stop new strategies from getting funded too, as the vault prioritizes minimumTotalIdle
```